assets/scss/components/_service-info.scss

```diff
@@ -68,30 +33,6 @@ Markup:
 Styleguide Service Info.With Help
 */
 
-
-/*
-Agent Name
-
-Service Info with `.service-info__agent-name` provides highlighting for `.acting-agent`
-
-Markup:
-<div class="service-info">
-  <div class="logo">
-    <span class="organisation-logo organisation-logo-medium">
-    HM Revenue &amp; Customs
-    </span>
-  </div>
-  <div class="service-info__agent-name">
-    <p class="agent-name">
-      <span>Agent Name</span>
-      <span class="acting-agent"><a href="#">link</a></span>
-    </p>
-  </div>
-</div>
-
-Styleguide Service Info.Agent Name
-*/
-
 /*
 Delegate
```

Comment:
This should also have been moved down with `Agent Name`. Sorry, didn't spot it before.